File: Programming Language(ver.2)/PHP/203.php
<?php
    $val = intval(fgets(STDIN));
    $array = array("", "one", "two", "three", "four");
    if($val <= 0 || $val > 4){
        printf("error\n");
    }else{
        printf("%s\n", $array[$val]);
    }
?>

File: Programming Language(ver.2)/C++/203.cpp
#include <iostream>
using namespace std;

int main()
{
    int num;
    cin >> num;
    switch (num)
    {
    case 1:
        cout << "one";
        break;
    case 2:
        cout << "two";
        break;
    case 3:
        cout << "three";
        break;
    case 4:
        cout << "four";
        break;
    default:
        cout << "error";
        break;
    }
    return (0);
}